dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -771,6 +771,15 @@ class StartupTest {
       const int maxFailures = 3;
       int currentFailures = 0;
       for (int i = 0; i < iterations; i += 1) {
+        // Startup should not take more than a few minutes. After 10 minutes,
+        // take a screenshot to help debug.
+        final Timer timer = Timer(const Duration(minutes: 10), () async {
+          print('Startup not completed within 10 minutes. Taking a screenshot...');
```

Comment:
10 minutes is basically just an arbitrary time I picked. Our CI times out at 30 minutes, and it can retest up to 3 times when it fails. Typically, it looks like it usually takes under 3 minutes.

Reply:
should we also log the filename? or is this gonna be obvious from the LUCI build output?

Reply:
It gets logged by the `screenshot` command. It'll print something like

```[STDOUT] stdout: Screenshot written to /path/to/screenshot_startup_2023-07-06T16:38:37.674817.png (683kB).```